### ISSUE TITLE
Bases with ongoing construction should not be able to Sleep

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#17]
```
Bases with ongoing construction (CPU, Reactor, Network, Security) should
not be able to Sleep; they are active.
```

Original issue reported on code.google.com by `funnyman3595` on 29 Apr 2008 at 5:57

**Comment by GoogleCodeExporter** [comment]
Original comment by `funnyman3595` on 29 Apr 2008 at 7:12
* Added labels: **Milestone-0.28**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git (95a0d25) and SVN (r589).
```

Original comment by `funnyman3595` on 30 Apr 2008 at 12:37
* Changed state: **Fixed**
